.github/workflows/ci.yml

```diff
@@ -200,8 +207,12 @@ jobs:
             {project}/tests/test_compat.py
             {project}/tests/test_igzip_lib.py
       - name: Build sdist
-        if: "runner.os == 'Linux'"
-        run: python setup.py sdist
+        if: "runner.os == 'Linux' and ${{ matrix.build_sdist }}"
+        run: bash -c 'pip install cython && python setup.py sdist'
```

Comment:
BTW, the modern way to build an sdist is with [build](https://github.com/pypa/build) with something like "pip install build && python -m build --sdist"

Reply:
Thanks for sharing! That is incorporated now!